ops/ccs-ops-misc/synthetic-data/scripts/synthea-automation/s3_utilities.py

```diff
@@ -11,27 +11,37 @@
 boto_config = Config(region_name="us-east-1")
 s3_client = boto3.client('s3', config=boto_config)
 
-# wait config for checking presence of manifest.xml in /Done folder
-# check very 30 seconds
+'''
+wait config for checking presence of manifest.xml in /Done folder; check very 30 seconds
+'''
```

Comment:
Not to be confusing but only the doc comments are in triple quotes, normal comments I think are still using # (Although python will still ignore these as comments as well)